tests/integration/utils.py

```diff
@@ -196,6 +196,30 @@ def make_list_of_packages_hashable(data):
     return sorted([[i["name"], i["type"], i["version"]] for i in data])
 
 
+def sort_pkgs_and_deps_in_place(packages=None, dependencies=None):
+    """
+    Sorts lists of packages and dependencies in place.
+
+    Sorting order: name -> version -> type
+
+    :param list packages: the list of packages
+    :param list dependencies: the list of dependencies
+    """
+
+    def sort_helper(elements):
```

Comment:
I am thinking if it would be beneficial to have this function outside of `sort_pkgs_and_deps_in_place`. Maybe it could be used by some tests in future, but we can move it if later if it will be needed. What do you think?

Reply:
Yeah, could be useful, I'd say it's fine to leave it here and only change it if it's needed